nerfstudio/configs/base_config.py

```diff
@@ -28,6 +28,7 @@
 
 warnings.filterwarnings("ignore", module="torchvision")
 
+
```

Comment:
Will do, but FYI I get these sort of extraneous diffs after running "black docs/ nerfstudio/ scripts/ tests/." I'll run it selectively only on the files that I touch but it would be nice if one could just run that one command and not have to check file-by-file in the future

Reply:
Running `ns-dev-test` should run black, isort, lint, tests

Reply:
my point is that running ns-dev-test causes changes in files that I'm not touching. For example, I see diffs such as this when running it on a fresh checkout of main:

<img width="723" alt="image" src="https://user-images.githubusercontent.com/868618/225116562-227f54d8-8779-44ce-9ddc-0d84a65a1ac7.png">

Furthermore, running pylint (on a fresh checkout) seems to fail:

<img width="1844" alt="image" src="https://user-images.githubusercontent.com/868618/225116865-538dc331-d268-4410-ad19-a384c9ef5a05.png">

Is there possibly something I'm doing wrong here?

Reply:
Have you run `pip install -e .[dev]`?

Reply:
hmm, I must have had the wrong versions of the libraries installed or something. things work now after reinstalling - sorry for the noise